generators/dart/text.js

```diff
@@ -92,8 +108,9 @@ Blockly.Dart['text_charAt'] = function(block) {
   // Get letter at index.
   // Note: Until January 2013 this block did not have the WHERE input.
   var where = block.getFieldValue('WHERE') || 'FROM_START';
-  var text = Blockly.Dart.valueToCode(block, 'VALUE',
-      Blockly.Dart.ORDER_UNARY_POSTFIX) || '\'\'';
+  var textOrder = (where == 'FIRST' | where == 'FROM_START') ?
```

Comment:
Is the single `|` instead of `||` intentional?

Reply:
Not intentional, good catch.